ui/imports/shared/popups/SettingsDirtyToastMessage.qml

```diff
@@ -0,0 +1,123 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtGraphicalEffects 1.13
+
+import utils 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+Rectangle {
+    id: root
+
+    property bool active: false
+    property bool saveChangesButtonEnabled: false
+
+    property Flickable flickable: null
```

Comment:
Could we somehow avoid this tight coupling? What if we want to derive some of the settings pages from ScrollView instead of Flikable?